e2e/main.py

```diff
@@ -302,11 +310,25 @@ def call_checker_endpoint(
             gas_costs[name] = int(ret["contents"][0]["gas_limit"])
             return ret
 
+        def get_tez_tokens_and_make_checker_an_operator(amnt):
+            call_endpoint(tez_wrapper, "deposit", (), amount=amnt)
+            update_operators = [
+                {
+                    "add_operator": {
+                        "owner": account,  # bob?
+                        "operator": checker.context.address,  # checker
+                        "token_id": 3,  # FIXME: How to determine this field?
```

Comment:
This is a FIXME in a few places actually. For kit I believe that we can pull it from `functions.json`, but we aren't currently producing anything like that for the wrapper.

Reply:
Yeah, that's a good point. I added a
```python
TEZ_TOKEN_ID = 2
```
at the top of the file like you had previously done, but eventually we should have a single source of truth, ideally the checker yaml file.